### DISCUSSION TITLE
xfrm_acq_expires setting interaction with StrongSwan's Android Client

### DISCUSSION THREAD

**Discussion by acrolink** [strongswan/strongswan#856] [Q&A] [answered]
In my SmartTV Android based systems I see this setting inside `init.rc`:

```
on boot:
       ...
       # IPsec SA default expiration length
       write /proc/sys/net/core/xfrm_acq_expires 3600
```

My question, is this setting optimized for the working of the StrongSwan's Android Client? Or does it need any tuning?

My aim is to stabilize the VPN connection between the Android client and IKEv2 server.

Thank you.

**Comment by tobiasbrunner** [comment, marked as answer]
It's completely unrelated because our Android app does not use the Linux kernel's IPsec stack at all (it contains its own IPsec implementation, libipsec).
